data/mods/Aftershock/items/armor/exosuit/exosuit_frame.json

```diff
@@ -59,6 +59,7 @@
         "rigid": true,
         "max_contains_volume": "4 L",
         "max_contains_weight": "15 kg",
+        "max_item_length": "500 cm",
```

Comment:
16ft?

Reply:
Ah oops! Set these extremely large for testing and didn't revert.